app/lib/components/noteElements/location_component.tsx

```diff
@@ -130,9 +130,7 @@ const LocationPicker: React.FC<LocationPickerProps> = ({
         <PopoverContent className="z-30">
           <div className="flex justify-center items-center w-96 h-96 bg-white shadow-lg rounded-md">
             <div className="absolute top-2 left-2 z-50">
-              <SearchBarMap onSearch={handleSearch} isLoaded={isLoaded !== null} onNotesSearch={function (searchText: string): void {
```

Comment:
This is a good start since the error is being thrown here. Your next step should be to check if the search bar on the map is implemented and connect it to here.